### ISSUE TITLE
Language switching / Translations

### ISSUE THREAD

**Issue by hybridherbst** [steveruizok/kdtype#11]
For kids that don't speak English but still want to type and read little words 🙂

Ideas: 
- "classic" - /de, /en etc as routes
- "let the browser do the work" - texts in HTML and make sure translation buttons work (no idea if this is technically even possible)
- "bring your own" - ?words=<list of words> or <link to list of words json> to pull in custom data without coding

Personally I like both the routes (for something built-in) and bring-your-own - e.g. could put in a personalized list of kids names in a kindergarten group to type these

**Comment by Ponjimon** [comment]
> "bring your own" - ?words= or to pull in custom data without coding

I do not think that this would be a good idea for a game aimed at kids. Can be easily abused by adding inappropriate words.

**Comment by hybridherbst** [comment]
Good point, even when I would certainly hope that kids at the age this is aimed at are under parental supervision.

**Comment by hybridherbst** [comment]
Fancy, seems there's actually already a profanity filter in place:
- https://github.com/steveruizok/kdtype/blob/0f44412d61b4b1d653f88c5879035a4c41eab2c3/src/game/utils.ts#L31

**Comment by steveruizok** [comment]
I like it! For internationalization, we would need to 

1. collect words from the target language
2. either connect a profanity filter for that language or verify that the word list is profanity-free
3. set up internationalization using something like react-intl, which we use on tldraw

Our best bet for words lists may be to look at wordle clones, given the popularity of that app.

**Comment by hybridherbst** [comment]
Here's one for german that looks reasonable, after looking at a number that just seemed to have random letter combinations or arcane old german words I have never heard of...
http://www.sobiki.de/3buchstaben.html. 
Love that it goes all the way to 37-letter words, german is quite a language 😁
